pysnooper/tracer.py

```diff
@@ -198,10 +198,15 @@ class Tracer:
 
     You can also use `max_variable_length=None` to never truncate them.
 
+    Print time in elapsed time format::
```

Comment:
"elapsed time format" is a confusing description. Explain more here.

Reply:
I'll fix this later.